src/pdf.js

```diff
@@ -34,7 +34,10 @@ if (typeof PDFJSDev === 'undefined' ||
   if (pdfjsSharedUtil.isNodeJS()) {
     var PDFNodeStream = require('./display/node_stream.js').PDFNodeStream;
     pdfjsDisplayAPI.setPDFNetworkStreamClass(PDFNodeStream);
-  } else {
+  } else if (typeof Response !== 'undefined' && 'body' in Response.prototype) {
+    var PDFFetchStream = require('./display/fetch_stream.js').PDFFetchStream;
+    pdfjsDisplayAPI.setPDFNetworkStreamClass(PDFFetchStream);
+   } else {
```

Comment:
@Rob--W do you know if we can enable *only* fetch_stream.js for Chrome extension?

Reply:
What do you mean?

1. Whether `fetch_stream` is disabled for every target except for the Chrome extension build, or
2. Whether `fetch_stream` can be unconditionally be used in the Chrome extension, and `display/network` be removed?

I can't answer 1, but if you meant 2:

- The streaming feature requires Chrome 43 - https://github.com/mozilla/pdf.js/issues/6126#issuecomment-130572300
- I looked at [the telemetry of the Chrome extension](https://github.com/Rob--W/pdfjs-telemetry) for August 2017 (this month), filtered by users with the latest version of the extension (1.8.557, released July 2017, 17th), with a user agent that contains Chrome/digits.

Based on 1.4M unique data points (i.e. pings where the client cannot be distinguished from another client in the same data set; this already happens when the user upgrades their browser version), the usage statistics are as follows:

```
Chrome 62 0.91%
Chrome 61 0.87%
Chrome 60 56.85% <-- Current stable (officially released July 2017, 25th)
Chrome 59 32.11%
Chrome 58 2.08%
Chrome 57 1.17%
Chrome 56 0.77%
Chrome 55 0.97%
Chrome 54 0.20%
Chrome 53 0.34%
Chrome 52 0.60%
Chrome 51 0.33%
Chrome 50 0.19%
Chrome 49 0.91% <-- last supported Chrome for Windows XP
Chrome 48 0.07%
Chrome 47 0.17%
Chrome 46 0.08%
Chrome 45 0.07%
Chrome 44 0.04%
Chrome 43 0.13% <-- First version where streams API is supported
Chrome 42 0.04%
Chrome 41 0.06%
Chrome 40 0.05%
Chrome 39 0.15%
Chrome 38 0.01%
Chrome 37 0.02%
Chrome 36 0.01%
Chrome 35 0.38% <-- mostly 35.0.1878.0, win7. Doesn't look like an official build.
Chrome 34 0.37% <-- Ubuntu https://apps.ubuntu.com/cat/applications/chromium-browser/
Chrome 33 0.02%
Chrome 32 0.01%
Chrome 31 0.01%
Chrome 100 0.01% <-- I don't know what this is...
```

1.13% of the users (16k pings) use a Chrome version that does not support the streams API (Chrome 42 and earlier). For now, I'd like to keep supporting these users.

Reply:
Thanks for response and analysis. I was asking about (2) -- fetch_stream can be unconditionally be used in the Chrome extension. You answered my question, so we will handle Chrome extension no different than generic viewer.